test/unit/pr_checker.test.js

```diff
@@ -193,6 +193,42 @@ describe('PRChecker', () => {
       assert(!status);
       assert.deepStrictEqual(logger.logs, expectedLogs);
     });
+
+    it('should skip wait check for Code & Learn PR', () => {
+      const logger = new TestLogger();
+
+      const expectedLogs = {
+        warn: [],
+        info: [['This is a Code & Learn PR, it doesn\'t have a time limit.']],
+        error: [],
+        trace: []
+      };
+
+      const now = new Date();
+      const youngPR = Object.assign({}, firstTimerPR, {
+        createdAt: '2017-10-27T14:25:41.682Z',
+        labels: {
+          nodes: [
+            {
+              name: 'code-and-learn'
+            }
+          ]
+        }
+      });
+
+      const checker = new PRChecker(logger, {
```

Comment:
@Tiriel just a suggestion move this JSON to its own file `code_and_learn_pr.json` in `test/fixtures` like every other files.

Reply:
I was talking about the other tests cases of this file, in fact, sorry, porr english ^^